script.js

```diff
@@ -0,0 +1,112 @@
+const game = { turn: 0 };
+const players = [{ score: 0 }, { score: 0 }];
+const diagonals = [
+  [
+    { row: 0, column: 0 },
+    { row: 1, column: 1 },
+    { row: 2, column: 2 },
+  ],
+  [
+    { row: 0, column: 2 },
+    { row: 1, column: 1 },
+    { row: 2, column: 0 },
+  ],
+];
+let cells;
+
+function getCellCoordinates(cell) {
+  const index = cells.indexOf(cell);
+  return { row: Math.floor(index / 3), column: index % 3 };
+}
+
+function initializeBoard() {
+  game.board = [
+    [null, null, null],
+    [null, null, null],
+    [null, null, null],
+  ];
+  registerCellClickListeners();
+}
+
+function updateBoard(coordinates) {
+  game.board[coordinates.row][coordinates.column] = game.turn;
+  game.changedCoordinates = coordinates;
+}
+
+function toggleTurn() {
+  game.turn = game.turn === 0 ? 1 : 0;
+}
+
+function recordVictory() {
+  players[game.turn].score += 1;
+}
+
+function getGameFirstTurnPlayer() {
+  return (players[0].score + players[1].score) % 2;
```

Comment:
Just realized that the scores can't serve as a proxy for total games since ties are a thing.